packages/spacecat-shared-http-utils/src/auth/auth-wrapper.js

```diff
@@ -48,6 +53,44 @@ export function authWrapper(fn, opts = {}) {
       return new Response('Unauthorized', { status: 401 });
     }
 
+    if (!context.auth) {
+      context.auth = {
+        hasScopes: async (scopes) => {
```

Comment:
To keep auth-wrapper focused, can the hasScopes implementation be moved to its own file?

I like the simple interface from a usability perspective (`hasScopes(scopes)`), so perhaps this function remains which calls the underlying impl in its own file? This would avoid having to pass additional args (or the entire context):

```
export async function hasScopes(scopes, apiKey) // where apiKey is the context.pathInfo?.headers['x-api-key'] val
```

Reply:
We need the dataAccess object as well for the apiKey query.